package.json

```diff
@@ -33,6 +33,7 @@
     "progress-bar-webpack-plugin": "^1.12.1",
     "source-map-loader": "^0.2.4",
     "ts-loader": "^5.3.3",
+    "tslint": "^5.12.1",
```

Comment:
@arudnicka Let's also add this line `"lint:fix": "tslint src/**/*.ts --fix"` for auto resolve issues in build time to `scripts`